rotkehlchen/globaldb/upgrades/v4_v5.py

```diff
@@ -62,29 +54,8 @@ def _populate_rpc_nodes(cursor: 'DBCursor', root_dir: Path) -> None:
 
 
 def _reset_curve_cache(write_cursor: 'DBCursor') -> None:
-    """Resets some parts of the curve cache.
-    1. Resets last queried timestamp to query and save gauges.
-    2. Clears curve pool tokens to requery them and save in the new format.
-    """
-    refreshing_timestamp = Timestamp(ts_now() - ETH_PROTOCOLS_CACHE_REFRESH - 1)
-    lp_tokens = globaldb_get_general_cache_values(
-        cursor=write_cursor,
-        key_parts=[GeneralCacheType.CURVE_LP_TOKENS],
-    )
-    if len(lp_tokens) == 0:
-        log.error('Expected curve pools cache to be populated, but it was not')
-        return
-
-    globaldb_set_general_cache_values_at_ts(
-        write_cursor=write_cursor,
-        key_parts=[GeneralCacheType.CURVE_LP_TOKENS],
-        values=lp_tokens,
-        timestamp=refreshing_timestamp,
-    )
-    globaldb_delete_general_cache_like(
-        write_cursor=write_cursor,
-        key_parts=[GeneralCacheType.CURVE_POOL_TOKENS],
-    )
+    """Resets curve cache to query gauges and update format of the lp tokens"""
```

Comment:
Since you are touching this file I notice it has multiple things missing.

1. The `migrate_to_v5` should have at its docstring the version where it's introduced. For v4->v5 this is 1.28.
2. At the end of `migrate_to_v5` the `Exit` debug log is missing